marimo/_server/ai/providers.py

```diff
@@ -180,6 +186,37 @@ def get_client(self, config: AiConfig) -> OpenAI:
         if not base_url:
             base_url = None
 
+        # add SSL parameters/values
+        ssl_verify: Optional[bool] = (
+            config.get("ai", {})
+            .get("open_ai", {})
```

Comment:
`config` is already `AiConfig`, so you don't need `config.get("ai", {})`

Also, we check `"open_ai" not in config`, so we can just pull this to re-use.

```
open_ai_config = config["open_ai"]

ssl_verify: bool = open_ai_config.get("ssl_verify", True)
...
```

and should clean up the code